packages/flutter/test/material/tabs_test.dart

```diff
@@ -1000,18 +1000,22 @@ void main() {
 
     // Not close enough to switch to page 2
     pageController.jumpTo(500.0);
+    await tester.pumpAndSettle();
```

Comment:
This was needed because of the changed condition in `_handleTabControllerIndexChangeStarted`. Previously, it always returned when `_warpUnderwayCount > 0`. But now we allow it to be interrupted when current warping index and new target index is different, so when `_controller.index = _pageController.page.round()` is called in `ScrollEndNotification`, `_warpToCurrentIndex` was called and it ran a meaningless `_pageController.animateToPage` once. This animation gets cancelled right away, so it doesn't do anything to UI but it kinda "dragged" a frame.

This is a bug because we actually don't need to run `_warpToCurrentIndex` in this case. So I changed the order of setting `_controller.index` and `_currentIndex` [here](https://github.com/flutter/flutter/pull/60080/commits/db8b31030d79446870a716718bf325c1ede9eec8) so that it wouldn't call unnecessary warping again.